src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -96,44 +117,40 @@ using namespace Containers::Literals;
 
 struct AssimpImporter::File {
     Containers::Optional<Containers::String> filePath;
+    UnsignedInt assimpVersion = 0;
     bool importerIsGltf = false;
     const aiScene* scene = nullptr;
     /* Index -> pointer, pointer -> index conversion for nodes as they're
        represented in a tree. Needed by objectCount() (which is const) so can't
        be delayed to scenne() parsing. */
-    std::vector<aiNode*> nodes;
+    Containers::Array<aiNode*> nodes;
     std::unordered_map<const aiNode*, UnsignedInt> nodeIndices;
     /* (materialPointer, propertyIndexInsideMaterial, imageIndex) tuple,
        imageIndex points to the (deduplicated) images array */
-    std::vector<std::tuple<const aiMaterial*, UnsignedInt, UnsignedInt>> textures;
+    Containers::Array<Containers::Triple<const aiMaterial*, UnsignedInt, UnsignedInt>> textures;
     /* (materialPointer, propertyIndexInsideMaterial) tuple defining the first
        (unique) location of an image */
-    std::vector<std::pair<const aiMaterial*, UnsignedInt>> images;
+    Containers::Array<Containers::Pair<const aiMaterial*, UnsignedInt>> images;
 
-    std::unordered_map<std::string, UnsignedInt> materialIndicesForName;
+    std::unordered_map<Containers::String, UnsignedInt> materialIndicesForName;
```

Comment:
Because this one takes a `String` instead of a `StringView` and STL maps are stupid, `materialForName()` should call `find(Containers::String::nullTerminatedView(name))` instead of `find(name)` to (hopefully) avoid an unnecessary string allocation on every lookup.

I really need to make my own hashmaps already :sweat_smile: